tests/Unit/DSC_CMForestDiscovery.tests.ps1

```diff
@@ -0,0 +1,259 @@
+        $scheduleConvertDays = @{
+            DayDuration    = 0
+            DaySpan        = 7
+            HourDuration   = 0
+            HourSpan       = 0
+            MinuteDuration = 0
+            MinuteSpan     = 0
+        }
+        $scheduleConvertDaysMismatch = @{
+            DayDuration    = 0
+            DaySpan        = 6
+            HourDuration   = 0
+            HourSpan       = 0
+            MinuteDuration = 0
+            MinuteSpan     = 0
+        }
+        $scheduleConvertHours = @{
+            DayDuration    = 0
+            DaySpan        = 0
+            HourDuration   = 0
+            HourSpan       = 7
+            MinuteDuration = 0
+            MinuteSpan     = 0
+        }
+        $standardGetDiscoveryOutput = @{
+            Props = @(
+                @{
+                    PropertyName = 'Startup Schedule'
+                    Value1       = '0001200000100038'
+                }
+                @{
+                    PropertyName = 'SETTINGS'
+                    Value1       = 'Active'
+                }
+                @{
+                    PropertyName = 'Enable AD Site Boundary Creation'
+                    Value        = 0
+                }
+                @{
+                    PropertyName = 'Enable Subnet Boundary Creation'
+                    Value        = 0
+                }
+            )
+        }
+        $standardGetInput = @{
+            SiteCode = 'Lab'
+            Enabled  = $true
+        }
+        $returnEnabledDaysMismatch = @{
+            SiteCode        = 'Lab'
+            Enabled         = $true
+            PollingSchedule = $mockCimPollingScheduleDayMismatch
+        }
+        $getReturnEnabledDays = @{
+            SiteCode                                  = 'Lab'
+            Enabled                                   = $true
+            PollingSchedule                           = $mockCimPollingSchedule
+            EnableActiveDirectorySiteBoundaryCreation = $true
+            EnableSubnetBoundaryCreation              = $true
+        }
+        $getReturnEnabledHours = @{
+            SiteCode        = 'Lab'
+            Enabled         = $true
+            PollingSchedule = $mockCimPollingScheduleHours
+        }
+        $getReturnDisabledOutput = @{
+            SiteCode                                  = 'Lab'
+            Enabled                                   = $false
+            PollingSchedule                           = $mockCimPollingSchedule
+            EnableActiveDirectorySiteBoundaryCreation = $false
+            EnableSubnetBoundaryCreation              = $false
+        }
+        $getInputDisableSubnet = @{
+            SiteCode                     = 'Lab'
+            Enabled                      = $true
+            EnableSubnetBoundaryCreation = $false
+        }
+        $getReturnDisabled = @{
+            SiteCode = 'Lab'
+            Enabled  = $false
+        }
+        Describe "$moduleResourceName\Get-TargetResource" {
```

Comment:
Take a look at other Unit tests and fixing spacing based on those.

Reply:
Fixed, based on the spacing in the other unit tests.